### PR TITLE
Feature/4129 Server property for default user creation

### LINKED ISSUE #4129: Server property to disable (or enable) default user creation
### Situation Description
Netzlink mentioned that they want to reduce the number of steps needed to setup SORMAS. One common step is to disable the default users (except admin).

### Feature Description
Add a property to `sormas.properties` which allows to control the creation of default users. Admin user should always be included. This  could be done by checking for the property in `StartupShutdownService.createDefaultUsers`.

### Possible Alternatives
Couple it with the SORMAS developer mode?

### PR DESCRIPTION
**Pull request by JonasCir**
closes #4129

### CODE REVIEW

File: sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java [hunk 1]
```diff
@@ -355,6 +355,15 @@ private void createDefaultUsers() {
 			userService.persist(admin);
 			userUpdateEvent.fire(new UserUpdateEvent(admin));
 
+
+			if (!configFacade.getCreateDefaultUsers()) {
```
Comment (MateStrysewske) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/common/StartupShutdownService.java:359]: Move this new code and the admin creation code to above the calls to retrieve regions, districts, communities etc. as that's unnecessary if no default users are created anyway

File: sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java [hunk 2]
```diff
@@ -534,7 +536,11 @@ public void validateAppUrls() {
 	}
 
 	public int getDashboardMapMarkerLimit() {
-		return getInt(DASHBOARD_MAP_MARKER_LIMIT, -1);
+	    return getInt(DASHBOARD_MAP_MARKER_LIMIT, -1);
+	}
+
+	public boolean getCreateDefaultUsers(){
```
Comment (MateStrysewske) [inline sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java:542]: boolean getters usually start with "is" instead of "get"

**Review by MateStrysewske — CHANGES_REQUESTED** [review]
Please use automatic code formatting (see development environment setup instructions), ConfigFacadeEjb.java is not properly formatted

File: sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java (hunk 2 above)
Comment (JonasCir) [reply sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java:542]: can we settle for should?

Comment (JonasCir) [reply sormas-backend/src/main/java/de/symeda/sormas/backend/common/ConfigFacadeEjb.java:542]: fixed

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**